homeassistant/components/ohme/config_flow.py

```diff
@@ -62,3 +74,38 @@ async def async_step_user(
         return self.async_show_form(
             step_id="user", data_schema=USER_SCHEMA, errors=errors
         )
+
+    async def async_step_reauth(
+        self, entry_data: Mapping[str, Any]
+    ) -> ConfigFlowResult:
+        """Handle re-authentication."""
+        return await self.async_step_reauth_confirm()
+
+    async def async_step_reauth_confirm(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle re-authentication confirmation."""
+        errors = {}
+        reauth_entry = self._get_reauth_entry()
+        if user_input is not None:
```

Comment:
I think we can just re-use the user step by returning `return await self.async_step_user(user_input)` on input and checking for `self.source == SOURCE_REAUTH` where necessary